pkg/controller/migration/convert/utils.go

```diff
@@ -0,0 +1,17 @@
+package convert
+
+import corev1 "k8s.io/api/core/v1"
+
+func getContainers(spec corev1.PodSpec, name string) *corev1.Container {
```

Comment:
This function only gets 1 container so I feel like the name should not be plural.